.github/workflows/ci.yml

```diff
@@ -0,0 +1,119 @@
+name: CI
```

Comment:
I don't have an opinion here, but I think we are basically using `Github Actions` and `CircleCI` to do the same thing. Both run the tests, the only difference is one does linting stuff and sends coverage.

It might be nice to just make use of Github Actions - if the pricing will be sufficient for this project.

Reply:
I don't have any idea of the pricing structure but I would remove CircleCI in the second PR.
And make use of only Github Action. 
In case of any issue we can rollback.

Reply:
On second thoughts, I just checked out: [managing Github actions billing](https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/viewing-your-github-actions-usage).
I would leave this open-ended unless needed or requested we can keep on using both CircleCI and Github Actions. 
Notice: I am not sure if billing applies to public repo but I would leave that to @BolajiAyodeji  for further information.